src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+
+interface TodoListProps {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  onDeleteTodo: (todoId: string) => void;
+}
+
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  tempTodo,
+  onDeleteTodo,
+}) => (
+  <section className="todoapp__main">
+    <TransitionGroup>
+      {todos.map((todo) => {
+        const { id, title, completed } = todo;
+
+        return (
+          <CSSTransition
+            key={id}
+            timeout={300}
+            classNames="item"
+          >
+            <div key={id} className={classNames('todo', { completed })}>
+              <label className="todo__status-label">
+                <input
+                  type="checkbox"
+                  className="todo__status"
+                  checked
+                  readOnly
+                />
+              </label>
+
+              <span className="todo__title">{title}</span>
+
+              <button
+                type="button"
+                className="todo__remove"
+                onClick={() => onDeleteTodo(String(todo.id))}
```

Comment:
I think you can just make this function accept a number instead of casting it to string